packages/x-charts/src/LineChart/AreaElement.tsx

```diff
@@ -10,6 +10,10 @@ import { InteractionContext } from '../context/InteractionProvider';
 export interface AreaElementClasses {
   /** Styles applied to the root element. */
   root: string;
+  /** Styles applied to the root element when higlighted. */
+  highlighted: string;
+  /** Styles applied to the root elementwhen fade out. */
+  fadeOut: string;
```

Comment:
IMHO, `fade-out` sounds like an animation to me. Maybe we could go with a less ambiguous word? 🤔 
What do you think about:
- emphasized/deemphasized
- emphasized/understated
- highlighted/opaque

Reply:
Come from this docs:
https://echarts.apache.org/en/option.html#series-bar.emphasis.focus

I have never seen the words `deemphasized` or `understated`

I don't see the problem that `fade-out` sounds like an animation since it would be the style applied at the end of a fade-out transition

After some dictionary browsing, I found `dismissed` in the sense that those elements are dismissed by the current selection state

Reply:
Well, you make some good points.
I got hung up on naming due to being unable to provide a more precise comment for this class. 🙈 🤷

Reply:
We lack a space by the way

```suggestion
  /** Styles applied to the root element twhen fade out. */
  fadeOut: string;
```

Concerning `fadeOut`, I'm not a huge fan but I don't find any alternative that much better :grimacing:

Reply:
We lack a space by the way

```suggestion
  /** Styles applied to the root element twhen fade out. */
  fadeOut: string;
```

Concerning `fadeOut`, I'm not a huge fan but I don't find any alternative that much better :grimacing:

Reply:
```suggestion
  /** Styles applied to the root element when it is faded out. */
  faded: string;
```
What do you think about this variant? 🤔

Reply:
That could be clearer indeed
`fade out` => the action to go from non-faded to fade
`faded` => the final state

Reply:
How is it called in our competitors packages ?

Reply:
Not a huge fan as well